pytorch/training/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -3,11 +3,11 @@ ARG PYTHON_VERSION=3.8.13
 ARG MAMBA_VERSION=4.12.0-2
 
 # PyTorch Binaries
-ARG PT_EC2_TRAINING_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.12.0_e3/20220714-051536/64a46e47aea7de3d99f75c5b4feba6d66c80af8b/torch-1.12.0%2Bcpu-cp38-cp38-linux_x86_64.whl
-ARG PT_SM_TRAINING_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.12.0_sm/20220713-224008/9f169f548f4b1eca6631c07d48bbe73a59e300ca/torch-1.12.0%2Bcpu-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHVISION_URL=https://download.pytorch.org/whl/cpu/torchvision-0.13.0%2Bcpu-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHAUDIO_URL=https://download.pytorch.org/whl/cpu/torchaudio-0.12.0%2Bcpu-cp38-cp38-linux_x86_64.whl
-ARG PT_TORCHDATA_URL=https://download.pytorch.org/whl/test/torchdata-0.4.0-cp38-cp38-linux_x86_64.whl
+ARG PT_EC2_TRAINING_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.12.1_e3/20220826-041141/579ffd0893a9d3e60773918bef0b000dfe60a7ce/torch-1.12.1%2Bcpu-cp38-cp38-linux_x86_64.whl
```

Comment:
We are no longer using the "e3" name and are now using "ec2" instead. Can you please change the name of these artifacts to reflect that change?